README.md

```diff
@@ -25,21 +25,25 @@ The required values can be found from the `Outputs` tab on your stack page creat
 
 Create *`custom.properties`* file inside the project root folder and add the details as below.
 
-KEY to be added in custom.properties | Corresponding Key from stack output |
---- | --- |
-DEFAULT_IDENTITY_POOL_ID | IdentityPoolId |
-DEFAULT_REGION | Take regin from IdentityPoolId (Cahracters before ':') |
-IDENTITY_POOL_ID | IdentityPoolId |
-USER_DOMAIN | UserDomain |
-USER_POOL_CLIENT_ID | UserPoolClientId |
-USER_POOL_ID | UserPoolId |
-WEB_SOCKET_URL | WebSocketUrl |
+KEY to be added in custom.properties | Corresponding Key from stack output                                   |
+--- |-----------------------------------------------------------------------|
+DEFAULT_IDENTITY_POOL_ID | IdentityPoolId                                                        |
+DEFAULT_REGION | Take region from IdentityPoolId (Character before ':')                |
+DEFAULT_SE_IDENTITY_POOL_ID | IdentityPoolId form ap-southeast-1 region for GrabMaps support        |
+DEFAULT_SE_REGION | Take region from ap-southeast-1 IdentityPoolId (Character before ':') |
+IDENTITY_POOL_ID | IdentityPoolId                                                        |
+USER_DOMAIN | UserDomain                                                            |
+USER_POOL_CLIENT_ID | UserPoolClientId                                                      |
+USER_POOL_ID | UserPoolId                                                            |
+WEB_SOCKET_URL | WebSocketUrl                                                          |
```

Comment:
this doesnt look good, please keep the original formatting here

Reply:
Updated